src/tools/ImportActivityReportDuration.js

```diff
@@ -0,0 +1,25 @@
+import {} from 'dotenv/config';
+import { option } from 'yargs';
+import importDuration from './importDuration';
+import { logger } from '../logger';
+
+const { argv } = option('file', {
+  alias: 'f',
+  description: 'Input .csv file',
+  type: 'string',
+}).option('region', {
```

Comment:
Can we remove this option if it isn't used?

Reply:
I have removed the region param.